assets/src/stories-editor/components/context-menu/index.js

```diff
@@ -181,9 +183,12 @@ const ContextMenu = ( props ) => {
 	};
 
 	let blockActions = [];
+	const pageList = getBlockOrder();
+	const numPages = pageList.length;
+	const isPage = isPageBlock( firstBlockClientId );
 
 	// Don't allow any actions other than pasting with Page.
```

Comment:
This comment should be updated too.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3537/commits/bbac22795430583fc9ba28c9e73953e90dedfcc0